modules/engines/engine-hdb/src/test/java/com/sap/xsk/hdb/ds/test/processors/XSKViewProcessorTest.java

```diff
@@ -71,18 +73,20 @@ public void openMocks() {
 
   @Test
   public void executeCreateViewHANAv1Successfully() throws Exception {
+
     XSKViewCreateProcessor processorSpy = spy(XSKViewCreateProcessor.class);
     String hdbviewSample = org.apache.commons.io.IOUtils
         .toString(XSKViewParserTest.class.getResourceAsStream("/ItemsByOrderHANAv1.hdbview"), StandardCharsets.UTF_8);
-
     XSKDataStructureHDBViewModel model = XSKDataStructureModelFactory.parseView("hdb_view.db/ItemsByOrderHANAv1.hdbview", hdbviewSample);
     model.setName("\"MYSCHEMA\".\"hdb_view::ItemsByOrderHANAv1\"");
+    model.setDbContentVersion(XSKDBContent.XS_CLASSIC);
     String mockSQL = "testSQLScript";
 
     //PowerMock do not support deep stub calls
     PowerMockito.mockStatic(SqlFactory.class, Configuration.class);
+
     when(SqlFactory.getNative(mockConnection)).thenReturn(mockSqlfactory);
-    when(SqlFactory.deriveDialect(mockConnection)).thenReturn(mockSqlDialect);
+    when(SqlFactory.deriveDialect(mockConnection)).thenReturn(new HanaSqlDialect());
```

Comment:
Why we use HanaSqlDialect instead of DefaultSqlDialect?

Reply:
This has been changed due to the different logic for checking the dialect type and the particular dialect should be passed to cover all of the scenarios-positive and negative, depending on the dialect.